app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -339,9 +339,13 @@ public static void openPlaylistFragment(FragmentManager fragmentManager,
                 .commit();
     }
 
-    public static void openWhatsNewFragment(FragmentManager fragmentManager) {
+    public static void openFeedFragment(FragmentManager fragmentManager) {
+        openFeedFragment(fragmentManager, -1, null);
```

Comment:
Maybe the `-1` constant (that means "show the whole feed, not that of a specific group") should probably be extracted to a constant for better readability.

Reply:
Done for all the files that use this constant.

Reply:
Good, thank you!